app/controllers/schools/participants_controller.rb

```diff
@@ -131,4 +140,28 @@ def participant_mentor_form_params
   def email_used?
     User.where.not(id: @profile.user.id).where(email: @profile.user.email).any?
   end
+
+  def active_participant_profiles
+    @school_cohort.active_ecf_participant_profiles
+  end
+
+  def participant_profiles
+    policy_scope(active_participant_profiles)
+  end
+
+  def ineligible_participants
+    policy_scope(active_participant_profiles.ineligible_status.includes(:user).order("users.full_name"))
```

Comment:
Why the school_cohort vs policy_scope here? Is there a chance 2020 participants (NQT+1) could show up where they aren't supposed to?

Reply:
I was following the original scope so thought this was something that might happen `@participant_profiles = policy_scope(
      @school_cohort.active_ecf_participant_profiles.includes(:user).order("users.full_name"),
    )`

Reply:
Oh yes, missed that method call